app/publish/spec.js

```diff
@@ -123,6 +123,16 @@ describe('publish', async() => {
         expect(result.message.toLowerCase()).to.contain('set tag');
     });
 
+    it('Should change tag if tagName specified', async() => {
+        NPM_FUNCTIONS.exists = () => true;
+        NPM_FUNCTIONS.publish = () => {};
+        NPM_FUNCTIONS.setTag = () => {};
```

Comment:
This could be nice as well
```suggestion
        NPM_FUNCTIONS.setTag = (package, version, tag) => expect(tag).to.equal('next');
```